planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -1317,6 +1312,7 @@ bool StartPlannerModule::planFreespacePath()
 void StartPlannerModule::setDrivableAreaInfo(BehaviorModuleOutput & output) const
 {
   if (status_.planner_type == PlannerType::FREESPACE) {
+    std::cerr << "Freespace planner updated drivable area." << std::endl;
```

Comment:
@kyoichi-sugahara you forgot remove tihs?

Reply:
@kosuke55 
Sorry I forgot to remove this and remove in this [PR](https://github.com/autowarefoundation/autoware.universe/pull/6187)